orderbook/src/api/cancel_order.rs

```diff
@@ -1,15 +1,26 @@
 use crate::api::extract_payload;
 use crate::orderbook::{OrderCancellationResult, Orderbook};
 use anyhow::Result;
-use model::order::OrderCancellation;
+use model::order::{OrderCancellation, OrderUid};
+use model::Signature;
+use serde::{Deserialize, Serialize};
 use std::{convert::Infallible, sync::Arc};
 use warp::{hyper::StatusCode, Filter, Rejection, Reply};
 
+#[derive(Deserialize, Serialize)]
+struct CancellationPayload {
```

Comment:
Only needs Deserialize, not Serialize.

Reply:
I think we need serialize:

```
 --> orderbook/src/api/cancel_order.rs:78:19
   |
78 |               .json(&CancellationPayload {
   |  ___________________^
79 | |                 signature: cancellation.signature,
80 | |             });
   | |_____________^ the trait `Serialize` is not implemented for `cancel_order::CancellationPayload`
```

Reply:
I forgot it was used in the test. It's fine if you want to re-add it. Alternatively you could run the test with a string containing the json directly.